src/components/anim.js

```diff
@@ -2,9 +2,9 @@
 
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { findDOMNode } from 'react-dom';
 import findKey from 'lodash/findKey';
 import { connect } from 'react-redux';
+import { findDOMNode } from 'react-dom';
```

Comment:
**Note:** There's only a diff here because I reordered these lib imports alphabetically

Reply:
if it is what you say it is, I love it